play-wright/e2e/individual-registration.spec.js

```diff
@@ -0,0 +1,57 @@
+import { test, expect  } from '@playwright/test';
+import { IndividualRegistrationPage } from '../pages/individual-registration.page'; 
+import { userLogin } from '../utils';
+const userDetails = require('../fixture/user-details.json');
+
+// Currently we are have implemented the registration with single user delete will need to
+// add functionality to delete user from admin dashboard or use faker library to generate new user in registration
```

Comment:
Currently implementation was done for using same user details for registration form, hence when check for existing user exist would have been implemented the test case would fail.